index.html

```diff
@@ -0,0 +1,208 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style/style.css" type="text/css">
+        <title>YouTube</title>
+    </head>
+    <body>
+        <!--Start of navigation bar-->
+        <header>
+            <nav>
+                <a href="#"><img id="bars" class="img" src="img/img_bar.png" alt="image bar"></a>
+                <a href="#"><img id="logo" src="img/youtube_logo.png" alt="YouTube"></a>
+                <input id="search" type="text" placeholder="Search" class="img">
+                <a href="#"><button id="loop"><img id="img1" src="img/img_loop.png" alt="image loop"></button></a>
+                <a href="#"><img id="upload" class="img" src="img/img_arrow.png" alt="image arrow"></a>
+                <a href="#"><img id="black_square" class="img" src="img/black_square.png" alt="square image"></a>
+                <a href="#"><img id="bell" class="img" src="img/bell.png" alt="bell image"></a>
+                <a href="#"><img id="profile" src="img/images.jpg" alt=""></a>
+            </nav>
+        <!--End of navigation bar-->
+        </header>
+
+        <!--Start of main page-->
+        <section id="main">
+            <div class="main-div">
+
+              <!--Begining of video section-->
+
+                <div class="left-div"><iframe id="video" width="900" height="480" src="https://www.youtube.com/embed/f9ip1iSAptQ" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                    <div class="video-info">
+                      <div id="p1">
+                          <p><h2>What is Microverse?</h2></p>
+                          10,084 views •Sep 19, 2019
+                      </div>
+                      <div id="center"><p></p></div>
+                      <div id="p2"> <br><br>
+                          <p>
+                              <a href="#"><img src="img/like.png" alt="Likes 236" height="16">&nbsp 236 &nbsp &nbsp</a>
+                              <a href="#"><img src="img/dislike.png" alt="Dislikes 4" height="16">&nbsp 4 &nbsp &nbsp</a>
+                              <a href="#"><img src="img/share.png" alt="Share" height="16px">&nbsp SHARE &nbsp &nbsp</a>
+                              <a href="#"><img src="img/save.png" alt="Save" height="16px">&nbsp SAVE</a>
+                          </p>
+                      </div>
+                    </div>
+                    <div id="channel-info">
+                        <img id="mic-logo" src="img/microverse.png" alt="Microverse" width="60" height="60"><p>&nbsp &nbsp Microverse </p>
+                        <div id="subscribe">
+                            <a href="#">
+                                <button id="button"><p>SUBSCRIBE</p></button>
```

Comment:
To fix this alignment, you do not need this <p> tag on line 51. You can just write SUBSCRIBE inside the <button> tag, like this: <button id="button">Subscribe</button> (you should not use an ID like this since you have different buttons on the page and the ID should be unique)